### ISSUE TITLE
using :min-clj-kondo-version results in incorrect warning

### ISSUE THREAD

**Issue by imrekoszo** [clj-kondo/clj-kondo#2463]
- [x] I have read the [Clojure etiquette](https://clojure.org/community/etiquette) and will respect it when communicating on this platform.

```
; clj-kondo -version
clj-kondo v2025.01.16
```

**platform**

native, macos, via homebrew

**problem**

`.clj-kondo/config.edn:1:2: warning: Linter config should go under :linters` warning is given when min version is specified in config.edn per #2362 

**repro**

```
; mkdir .clj-kondo
; echo '{:min-clj-kondo-version "2025.01.16"}' > .clj-kondo/config.edn
; clj-kondo --lint .clj-kondo/config.edn
.clj-kondo/config.edn:1:2: warning: Linter config should go under :linters
linting took 43ms, errors: 0, warnings: 1
```

**config**

```clojure
{:min-clj-kondo-version "2025.01.16"}
```

**expected behavior**

No warning

**Comment by imrekoszo** [comment]
Proposed a fix in #2480
